### PR TITLE
FFWEB-1248 be more specific about optional and required templates

### PR DESCRIPTION
**Pull request by torsten-knauf**

### CODE REVIEW

File: markdown/3.x/en/ff-asn.md [hunk 1]
```diff
@@ -147,9 +147,11 @@ Slider groups are handled in a different way. To style them you need to use the
 <ff-asn-group-slider></ff-asn-group-slider>
 ```
 
-A slider can have a `[slot="groupCaption"]` attribute but instead of detailedLinks and hiddenLinks the slider group accepts an `ff-slider-control` and/or an `ff-slider` element.
+A slider can have a `[slot="groupCaption"]` attribute and a `[data-container="removeFilter"]` element but instead of `detailedLinks` and `hiddenLinks` the slider group accepts an `ff-slider-control`.
```
Comment (wdutkiewicz-obj) [inline markdown/3.x/en/ff-asn.md]: Is it mandatory to have both `[slot="groupCaption]` and `[data-container="removeFilter"]` in the same time to replace `detailedLinks` and `hiddenLinks` with `ff-slider-control`? I think they're both optional, but when reading this sentence I understood this like: 'you can replace 'detailedLinks' and 'hiddenLinks' with 'ff-slider-control' when both `[slot="groupCaption"]` attribute and `[data-container="removeFilter"]` are present'. Please, try to express this in a way that does not allow similar interpretations.

File: markdown/3.x/en/ff-asn.md [hunk 2]
```diff
@@ -147,9 +147,11 @@ Slider groups are handled in a different way. To style them you need to use the
...
 
-The `ff-slider-control` can have two `input` elements annotated with the `[data-control="1/2"]` attribute. The input elements automatically react to user input (enter key pressed) and start to filter. You can change the behavior of the controls by using the appropriate attributes as described in `ff-slider-control-section`'s [API documentation](/api/3.x/ff-asn#tab=api)
+If no templates are provided for the `ff-slider-control` the templates default to the example below. If you want to customize the templates you must provide an `ff-slider` and an `input` element with attribute `data-control='1'` as well one with `data-control='2'`.
```
Comment (wdutkiewicz-obj) [inline markdown/3.x/en/ff-asn.md]:
```suggestion
If no templates are provided for the `ff-slider-control`, the templates default to the example below. If you want to customize the templates, you must provide an `ff-slider` and an `input` element with attribute `data-control='1'` as well as one with `data-control='2'`. 
```

File: markdown/3.x/en/ff-asn.md [hunk 3]
```diff
@@ -147,9 +147,11 @@ Slider groups are handled in a different way. To style them you need to use the
...
+A slider can have a `[slot="groupCaption"]` attribute and a `[data-container="removeFilter"]` element but instead of `detailedLinks` and `hiddenLinks` the slider group accepts an `ff-slider-control`.
 
-The `ff-slider-control` can have two `input` elements annotated with the `[data-control="1/2"]` attribute. The input elements automatically react to user input (enter key pressed) and start to filter. You can change the behavior of the controls by using the appropriate attributes as described in `ff-slider-control-section`'s [API documentation](/api/3.x/ff-asn#tab=api)
+If no templates are provided for the `ff-slider-control` the templates default to the example below. If you want to customize the templates you must provide an `ff-slider` and an `input` element with attribute `data-control='1'` as well one with `data-control='2'`. 
+
+The input elements automatically react to user input (enter key pressed) and start to filter. You can change the behavior of the controls by using the appropriate attributes as described in `ff-slider-control-section`'s [API documentation](/api/3.x/ff-asn#tab=api)
```
Comment (wdutkiewicz-obj) [inline markdown/3.x/en/ff-asn.md]:
```suggestion
The input elements automatically react to user input ('Enter' key pressed) and start filtering. You can change the behavior of the controls by using the appropriate attributes as described in `ff-slider-control` section's [API documentation](/api/3.x/ff-asn#tab=api)
```

File: markdown/3.x/en/ff-asn.md [hunk 4]
```diff
@@ -158,17 +160,19 @@ The `ff-slider-control` can have two `input` elements annotated with the `[data-
...
+           <ff-slider step-size="1" submit-on-release="true">
+               <div slot="slider1"></div>
+               <div slot="slider2"></div>
+           </ff-slider>
+           <div slot="sliderControls" style="display: flex; justify-content: space-around; align-items: center">
+               <input data-control="1" style="width: 60px;">
+               <span style="width: 20px; height: 2px; background-color: black; display: inline-block"></span>
+               <input data-control="2" style="width: 60px;">
+            </div>
+       </div>
```
Comment (wdutkiewicz-obj) [inline markdown/3.x/en/ff-asn.md]: Was this additional `div` wrapper necessary to make it working? I remember there was some change in ff-slider-control related to this - just want to make sure why you changed the documentation here. And why added additional style to `sliderControls`? Did you just check if this example work on our controls? If so, it's OK then.

Comment (torsten-knauf) [reply markdown/3.x/en/ff-asn.md]: yeah it worked before and works now as well.

Now the shown template is exactly the default template. I thought it is a nice easy extra information to write _"the templates default to the example below."_, as I added it.

File: markdown/3.x/en/ff-asn.md (hunk 1 above)
Comment (torsten-knauf) [inline markdown/3.x/en/ff-asn.md]:
```suggestion
A slider can have a `[slot="groupCaption"]` attribute, a `[data-container="removeFilter"]` element and a `ff-slider-control` element.
```

Comment (torsten-knauf) [reply markdown/3.x/en/ff-asn.md]: As it is already stated above, that the slider works differently, I think mentioning elements which are not supported is only confusing. What do you think about [my suggestion below](https://github.com/FACT-Finder-Web-Components/documentation/pull/195/files#r294187097)?

File: markdown/3.x/en/ff-asn.md (hunk 4 above)
Comment (wdutkiewicz-obj) [reply markdown/3.x/en/ff-asn.md]: Ah, I see now. I missed the context.

File: markdown/3.x/en/ff-asn.md (hunk 1 above)
Comment (wdutkiewicz-obj) [reply markdown/3.x/en/ff-asn.md]: I'm confused with `a` and `an` usage before `ff-*`. As far as I know we use `a` before words starting with consonants, `an` before words starting with vowels and whose pronounce like vowel-started (like "an hour"). Which is correct here: "a ff-component" or "an ff-component". The later one suits better for me.

Comment (torsten-knauf) [reply markdown/3.x/en/ff-asn.md]: I am totally with you here 👍

**Pull request merged**